.github/workflows/node.js.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
```

Comment:
This is a breaking change - you should upgrade your package version to the next major version in this case and clearly document it.

Is it necessary to drop the Node 12 support?

If possible, maybe create two new versions - one that simply supports gremlin 3.6.1 and fixes it (patch) and one that upgrades all other dependencies while dropping Node 12 (major)?

The major upgrade is more important for vulnerability fixes, but if someone is using Node 12 they're essentially stuck until they upgrade.

Reply:
Node 12 has reached its EOL on April 2022, the package will still work with it, but the tests will not. https://nodejs.org/en/about/releases/

People concerned by vulnerabilities should really update their node version.